README.md

```diff
@@ -82,48 +93,23 @@ For more examples, look at the
 suite of tests under kola. These tests were ported into kola and make
 heavy use of the native code interface.
 
-## ore
-
-Related commands to launch instances on Google Compute Engine(gce)
-within the latest SDK image. SSH keys should be added to the gce project
-metadata before launching a cluster. All commands have flags that can
-overwrite the default project, bucket, and other settings.  `ore help
-<command>` can be used to discover all the switches.
-
-### ore upload
-
-Upload latest SDK image to Google Storage and then create a gce image.
-Assumes an image packaged with the flag `--format=gce` is present.
-Common usage for CoreOS devs using the default bucket and project is:
+### Manhole
+The `platform.Manhole()` function creates an interactive SSH session which can
+be used to inspect a machine during a test.
 
-`ore upload`
-
-### ore list-images
-
-Print out gce images from a project. Common usage:
-
-`ore list-images`
-
-### ore create-instances
-
-Launch instances on gce. SSH keys should be added to the metadata
-section of the gce developers console. Common usage:
-
-`ore create-instances -n=3 -image=<gce image name> -config=<cloud config file>`
-
-### ore list-instances
-
-List running gce instances. Common usage:
-
-`ore list-instances`
+## ore
 
-### ore destroy-instances
+Related commands to launch instances on a variety of plaforms (gcloud, aws,
+azure, esx, and packet) within the latest SDK image. ore is mimics the underlying
```

Comment:
nit, `ore is` is a typo. I'll suggest `ore provides a low level interface for each cloud provider`